js/src/components/SchemaSettings.js

```diff
@@ -26,6 +26,15 @@ class SchemaSettings extends Component {
 		// Save the initial value for the alert check.
 		this.initialPageType = props.pageType.value;
 		this.initialArticleType = props.articleType ? props.articleType.value : "none";
```

Comment:
Not really related to your PR, but I see we forgot to rename this `none` to `None`. Bonus points? :)